psydac/ddm/cart.py

```diff
@@ -34,103 +34,89 @@ def find_mpi_type( dtype ):
 
     return mpi_type
 
-#====================================================================================
-class MultiCartDecomposition:
+class MultiPatchDomainDecomposition:
```

Comment:
Can you please list all classes in this file in the `__all__` variable on top?

Reply:
done in [7a503df](https://github.com/pyccel/psydac/pull/232/commits/7a503df0a47675b54ec389d5009fc80386b3f766)